src/types/vault.ts

```diff
@@ -0,0 +1,46 @@
+import { PolkadotAmount, BTCAmount } from "@interlay/monetary-js";
+import { AccountId } from "@polkadot/types/interfaces";
+import * as interbtcIndex from "@interlay/interbtc-index-client";
+
+import { VaultStatus } from "../interfaces";
+import Big from "big.js";
+
+export interface WalletExt {
+    // network encoded btc addresses
+    publicKey: string;
+    btcAddress?: string;
```

Comment:
I think this can be removed, most requests will specify the `btcAddress` for payments.